libafl/src/feedbacks/new_hash_feedback.rs

```diff
@@ -123,18 +126,24 @@ where
             .get_mut::<NewHashFeedbackMetadata>(&self.name)
             .unwrap();
 
-        match observer.hash() {
-            Some(hash) => {
-                let res = backtrace_state
-                    .update_hash_set(hash)
-                    .expect("Failed to update the hash state");
-                Ok(res)
-            }
+        let res = match observer.hash() {
+            Some(hash) => backtrace_state
+                .update_hash_set(hash)
+                .expect("Failed to update the hash state"),
```

Comment:
I mean, while we're at it: This should return an err (sorry for touching this line :D)